.gitignore

```diff
@@ -44,6 +45,9 @@ deps/cache
 [._]*.sw[a-p]
 [._]sw[a-p]
 
+# emacs stuff
```

Comment:
also `vim`

Reply:
I guess this could be part of the IDE section

Reply:
I don't understand what you mean - please elaborate. Or better just commit on top. I mostly care that ignored files are ingored. What section they are in or additional comments are great, but I don't have a feel for that. 

Right now there is: 

```
# vim stuff
[._]*.sw[a-p]
[._]sw[a-p]

# emacs stuff
*~

# IDE files
...
```
What would you have instead?

Reply:
Oh ok, missed the vim section. I meant that vim might also save `*~` files, so the line you added is also related to vim. But yea, not too important